app/forms/personal_key_form.rb

```diff
@@ -27,4 +31,9 @@ def extra_analytics_attributes
       multi_factor_auth_method: 'personal key',
     }
   end
+
+  # Reset sensitive fields so they don't get sent back to the browser
```

Comment:
If it were me, I'd name this method `reset_sensitive_fields` rather than having the comment. 2 for the price of 1!